buildtest/menu/build.py

```diff
@@ -148,13 +151,32 @@ def func_build_subcmd(args):
     # check user's buildtest setting for any errors by validating against settings schema
     check_settings(settings_file)
 
+    config_paths_testdir = config_opts.get("config", {}).get("paths", {}).get("testdir")
+    config_paths_buildspec_path = (
+        config_opts.get("config", {}).get("paths", {}).get("buildspec_path")
+    )
+    buildspec_paths = []
+    if config_paths_buildspec_path:
+        # return a unique list of directory paths where buildtest should search for buildspecs
+        # this is equivalent to $PATH for buildspec
+        buildspec_paths = list(set(config_paths_buildspec_path.split(":")))
```

Comment:
No need to strip for empty spaces (e.g., if there is a trailing : you are going to get an empty string as a path which will have unintended consequences...

Reply:
I see the trailing `:` is an issue i can put a check and remove it if found at end of char, we okay with splitting by colon i assume since this check would be done by default.

Reply:
i tested this locally and got this resolved, see latest commit